packages/relay/package.json

```diff
@@ -17,5 +17,9 @@
   "devDependencies": {
     "@types/react-relay": "14.1.4",
     "@types/relay-runtime": "14.1.10"
+  },
+  "peerDependencies": {
+    "react": ">=18.0.0",
+    "react-dom": ">=18.0.0"
```

Comment:
why?

Reply:
so we can use as import and the places we use knows this is a dependency.

Reply:
it does not scream right now because we have many packages installed at the base which needs to be placed differently. I've to handle that seperately.